Formula/jpeg-xl.rb

```diff
@@ -0,0 +1,32 @@
+class JpegXl < Formula
+  desc "New file format for still image compression"
+  homepage "https://jpeg.org/jpegxl/index.html"
+  url "https://gitlab.com/wg1/jpeg-xl.git", tag: "v0.2", revision: "31c71b0f61123a40789b0b8f54feb70e5995420e"
+  license "Apache-2.0"
+
+  depends_on "cmake" => :build
+  depends_on "gcc"
+  depends_on "giflib"
+  depends_on "ilmbase"
+  depends_on "jpeg"
+  depends_on "libpng"
+  depends_on "openexr"
+
+  fails_with :clang
+
+  def install
+    mkdir "build" do
+      system "cmake", "-DCMAKE_BUILD_TYPE=Release", "-DBUILD_TESTING=OFF", ".."
```

Comment:
```suggestion
      system "cmake", "-DCMAKE_BUILD_TYPE=Release", "-DBUILD_TESTING=OFF", "..", *std_cmake_args
```
Why not use `std_cmake_args`?

Reply:
These commands are from the [JPEG-XL repository](https://gitlab.com/wg1/jpeg-xl).  I'm happy to try your suggestions though.

Reply:
`std_cmake_args` specifies a `Release` build (among other things), so this should work fine.